webrtc.html

```diff
@@ -6570,9 +6594,22 @@ <h2>Attributes</h2>
               <code><a>RTCRtpReceiver</a></code> object <var>receiver</var>.
               When one of the SSRCs for RTP source media streams received
               by <var>receiver</var> is removed (either due to reception of
-              a BYE or via timeout), the <code>mute</code> event is fired at
-              <code>track</code>.  If and when packets are received again,
-              the <code>unmute</code> event is fired at <code>track</code>.</p>
+              a BYE or via timeout), <a>set the muted state</a> of the
+              <code>track</code> to the value <code>true</code>.
```

Comment:
Yes, that's something I was thinking of. Though, 10.3 needs some updates since it is just mentioning PeerConnection (and has not been updated to reflect the introduction of receiver and transceiver).

Also, 'RTP source' is undefined.